make/ide/xcode/hotspot/CreateXcodeProject.gmk

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (c) 2023, 2024, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+# This must be the first rule
+default: all
+
+include $(SPEC)
+include MakeBase.gmk
+include CopyFiles.gmk
+include Execute.gmk
+include JavaCompilation.gmk
+
+ifeq ($(call isTargetOs, macosx), true)
+  ##############################################################################
+  # Build the XcodeProjectMaker java tool.
+
+  PROJECT_MAKER_DIR := $(TOPDIR)/make/ide/xcode/hotspot
+  TOOLS_OUTPUTDIR := $(MAKESUPPORT_OUTPUTDIR)/ide/xcode
```

Comment:
I would have expected this to go in `$(BUILDTOOLS_OUTPUTDIR)`. Not that it matters that much. It only really affects which clean target removes what.

Reply:
For me, buildtools means "binaries we create to help with the build of the product" (basically specialized compilers).

Generating an IDE project is a different kind of story. Given your question, I checked what the Visual Studio java tool did. It created it's output in `$(HOTSPOT_OUTPUTDIR)/support/ide_classes`. That's not really better. (I think it should go to `$(MAKESUPPORT_OUTPUTDIR)/ide/visualstudio`.)